.travis.yml

```diff
@@ -7,6 +7,16 @@ go:
   - 1.14
 os: osx
 
+cache:
+  directories:
+    - /usr/local
```

Comment:
> GoTorch, in particular, gotorch/vision/transforms, uses OpenCV for image data augmentation. In Travis CI macOS VMs, we need to install OpenCV and its many dependencies. This would trigger Homebrew to update itself.  Thsi updating would take about 20 minutes. To shrink the time cost, we make Travis CI to cache the /usr/local directory, where Homebrew installs software packages.

Reply:
Done, add the comment.